style.css

```diff
@@ -228,37 +228,45 @@ body {
 .sponsor-title {
   margin-top: 10%;
   text-align: center;
+  color: #fff;
 }
 
 .sponsor-img {
   margin-top: 5%;
-  width: 25rem;
+  width: 5rem;
   list-style-type: none;
+  opacity: 100%;
+}
+
+.div-sponsor {
+  background-color: #2e2e2e;
 }
 
 .sponsor {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
+  flex-wrap: wrap;
   align-items: center;
   padding: 5%;
-  gap: 8rem;
+  gap: 1rem;
+  margin-left: 10%;
 }
 
 .footer {
   display: flex;
   justify-content: space-between;
   gap: 20px;
-  background-color: #2e2e2e;
+  background-color: #fff;
   padding: 10%;
 }
```

Comment:
- [ ] The about page does not have a partner section in the desktop view and instead has a dark footer. Kindly address this issue 😄 
- Your design:
![Screenshot 2022-07-02 054651](https://user-images.githubusercontent.com/85113567/176986891-c151f768-3692-4dea-9f50-df005bced019.png)
- Required design:
![Screenshot 2022-07-02 054716](https://user-images.githubusercontent.com/85113567/176986896-132d309f-982b-4282-93a0-fba16a901dd7.png)